tools/configure_travis.sh

```diff
@@ -2,6 +2,16 @@
 
 set -exu
 
+if [ "${TRAVIS_BUILD_TYPE}" = "Release" -a "${CXX}" = "g++-4.9" ]; then
+  # do a pgo build.  for this we do pretty terrible stuff.  make two subdirectories, do
+  # the pgo build + training in one.
+  mkdir ../pyston-build-pgo && cd ../pyston-build-pgo
+  cmake -GNinja -DTEST_THREADS=4 $TRAVIS_BUILD_DIR -DCMAKE_BUILD_TYPE=Release -DENABLE_PGO=ON -DPROFILE_STATE=generate -DPROFILE_DIR=`pwd`/../pyston-build
+  ninja -j4 pyston check-deps
+  ./pyston ${TRAVIS_BUILD_DIR}/minibenchmarks/combined.py
+  cd ../pyston-build
+fi
```

Comment:
yeah.  It's something that would be nicer to do more often than just when spinning a release (just in case things break), but it's definitely a pretty big hit.